DZ_5/DZ_5_1.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <style>
+    td.black{
+        background-color: black;
+        border: 1px solid blue;
+    }
+    
+    
+    td.figures1{
+        color: red;
+    }
+    td.figures2{
+        color:blueviolet;
+    }
+
+    </style>
+</head>
+<body>
+   <div id = "board"> </div> 
+                       
+    <script>
+    function chess(){
+        var $table = document.createElement('table');
+        var $board = document.getElementById('board');
+        var $liter =[ '','A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
```

Comment:
Здесь не нужно $ т.к. переменная хранит массив, который с DOM никак не связан. С двумя перменными строчкой ниже ситуация та же
```suggestion
        var liter = ['','A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
```